### DISCUSSION TITLE
Recommerce buyback solution

### DISCUSSION THREAD

**Discussion by JuHwon** [saleor/saleor#11167] [Q&A]
Hello, i am new to saleor and i just checked out some of the developer docs.
I am wondering if it is possible to achieve a "buyback" solution with saleor. 
Current usecase: An ecommerce shop who usually sells goods, want's to also provide a solution for their customers so they can return items after usage. e.g. the customer returns an item after 3 years with partial money back. We distinguish trade-in and buy-back. In both cases the money will go to the customer when the goods receive at the dealers facility. Would something like this be possible with saleor?

**Comment by CossackDex** [comment]
Hi @JuHwon 
I'm afraid that's currently impossible with pure Saleor, but you can try to build an app that will extend Saleor possibilities by described use case. You can find more details about building apps here https://docs.saleor.io/docs/3.x/developer/extending/apps/key-concepts

**Comment by timuric** [comment]
@JuHwon Seems like refunds is what you are looking for?

**Reply by JuHwon** [reply]
@timuric thanks. Refunds usually are only possible to the same payment method the user purchased with, right?
This could be a problem when the time between purchase and refund is e.g. 3 years.

**Reply by timuric** [reply]
@JuHwon in this case you are looking into sending money to customers bank account, sounds like Saleor is not the right place to solve this problem
